src/resources/views/character/partials/contact-links.blade.php

```diff
@@ -0,0 +1,38 @@
+@if (!in_array($row->contact_type, ['corporation', 'alliance']))
```

Comment:
An interesting approach. Is there a way you make it a bit more generic, so it can be used everywhere we use those links ribbon (ie: in intel, summary sidebar, etc...)

Reply:
i just updated the PR according to your feedback: 
Now the view lies in partials folder, no longer in /characters/partials: you might want to use `return view('web::partials.links', ['id' => $row->contact_id, 'type' => $row->contact_type]);`